cargo-shuttle/src/lib.rs

```diff
@@ -486,18 +489,78 @@ impl Shuttle {
     }
 
     async fn project_create(&self, client: &Client) -> Result<()> {
-        let project = client.create_project(self.ctx.project_name()).await?;
+        self.wait_with_spinner(
+            &[project::State::Ready],
```

Comment:
Should include `Errored` as well, as one of the potential "terminal" states. This could actually happen on create if, say, we run into resource constraints.